profiler/test/Datadog.Profiler.IntegrationTests/SmokeTests/SmokeTestRunner.cs

```diff
@@ -26,10 +26,10 @@ public SmokeTestRunner(string appName, string framework, string appAssembly, ITe
         {
         }
 
-        public SmokeTestRunner(string appName, string framework, string appAssembly, string commandLine, ITestOutputHelper output, bool enabledNewPipeline = false)
+        public SmokeTestRunner(string appName, string framework, string appAssembly, string commandLine, ITestOutputHelper output, bool enableNewPipeline = false)
```

Comment:
Shouldn't we test with the new pipeline by default?

Reply:
not needed since we will soon remove the code